### DISCUSSION TITLE
Configure SSL certificate auto-renewal

### DISCUSSION THREAD

**Discussion by aidanhmiles** [mozilla/hubs#3237] [Hubs Cloud Questions] [answered]
Hi there,

Is it possible to reconfigure when Let's Encrypt auto-renews the SSL cert for my Hub? I'm struggling to find documentation on the Hubs website or in Discord about this.

We set up a functioning self-hosted Hub about 2.5 months ago, and recently I started getting Lets Encrypt notices emailed to me related to SSL certs. Some of these alerts are related to subdomains from EC2 instances that are now spun down, and one is related to the currently active Hub EC2 instance.

I'd like to be able to configure auto-renewal in the way that Let's Encrypt suggests (renewing a cert 30 days before expiry). Is this supposed to be possible, or am I supposed to just let Hub take care of it?

**Comment by robinkwilson** [comment, marked as answer]
Hubs Cloud is configured to take care of it automatically for you. No extra steps needed on your end. 

(Unless you followed Recipe 3 and generated your own SSL certificates, then you'd manually renew them every year as the template and documentation says).

**Reply by robinkwilson** [reply]
(marking to get this question out of the pipeline, if this doesn't answer your question, click "unmark as answer")

**Reply by aidanhmiles** [reply]
This mostly answers the question, but it may be worth making it explicit in the docs somewhere. I only say that because I got a specific request from a supervisor (who isn't as trusting of automation as the engineers) to have it renew several days in advance the renewal deadline, "just in case". Thanks again for being so responsive!
